src/features/klant/persoon/PersoonZoeker.vue

```diff
@@ -190,16 +189,12 @@ const singleBsn = computed(() => {
 });
 
 const router = useRouter();
-const organisatieIds = useOrganisatieIds();
-watch(
-  [singleBsn, () => organisatieIds.value[0]],
-  async ([bsn, org], [oldBsn]) => {
-    if (org && bsn && bsn !== oldBsn) {
-      const { id } = await ensureKlantForBsn({ bsn }, org);
-      await router.push(`/personen/${id}`);
-    }
-  },
-);
+watch(singleBsn, async (bsn, oldBsn) => {
```

Comment:
enig idee waarom hier in de oude code orgId's gewacthced werden?

Reply:
Besproken: in open klant 1 moet je de bronorganisatie meegeven als je een klant of een contactmoment aanmaakt. In open klant 2 hoeft dit niet. We gaan overleggen of dit een functionaliteit is die aan open klant 2 toegevoegd moet/gaat worden.